pkg/lib/authn/mfa/recovery_code.go

```diff
@@ -24,3 +24,12 @@ func GenerateRecoveryCode() string {
 	code := rand.StringWithAlphabet(recoveryCodeLength, base32.Alphabet, rand.SecureRand)
 	return code
 }
+
+func NormalizeRecoveryCode(code string) (normalized string, err error) {
+	normalized, err = base32.Normalize(code)
+	if err != nil {
+		err = ErrRecoveryCodeNotFound
```

Comment:
It feels weird that a normalization function returns a NotFound error. Personally I'd prefer to return the error as-is, and translate any normalization error to NotFound error in `GetRecoveryCode`.